src/main/java/com/minecolonies/util/LanguageHandler.java

```diff
@@ -62,8 +63,10 @@ public static String getString(String key)
      * @param defaultValue the value to return if no key is found
      * @return Localized string
      */
+    @SuppressWarnings("deprecation")
```

Comment:
I would just leave the deprecations warnings. That we we're remembered that it's deprecated.
Or does it influence anything badly?
We can add a comment over it explaining why we leave it.